akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -132,6 +152,16 @@ class CircuitBreaker(
     new CircuitBreaker(scheduler, maxFailures, callTimeout, resetTimeout, maxResetTimeout, 2.0)(executor)
   }
 
+  /**
+   * The `resetTimeout` will be increased exponentially for each failed attempt to close the circuit.
+   * The default exponential backoff factor is 2.
+   *
+   * @param maxResetTimeout the upper bound of resetTimeout
+   */
+  def withExponentialBackoff(maxResetTimeout: java.time.Duration): CircuitBreaker = {
+    new CircuitBreaker(scheduler, maxFailures, callTimeout, resetTimeout, maxResetTimeout.asScala, 2.0)(executor)
```

Comment:
^

Reply:
ok, it is also an option and i will do it